incluster_eventing/emitter/config.yaml

```diff
@@ -0,0 +1,14 @@
+name: event-emitter
+namespace: default
+runtime: nodejs14
+source:
+    sourceType: inline
+apiRules:
+    - name: incomming-http-trigger
+      service:
+        host: incomming.inclusterevent.wookiee.shoot.canary.k8s-hana.ondemand.com
```

Comment:
```suggestion
        host: incomming.{put your runtime domain here}
```

Reply:
@strekm is it possible for api gateway to resolve domain when definind API rules?
Could I just pass subdomain name? ( i.e `incoming`) and api-gateway would resolve the whole URL?

Reply:
when passing only short hostname, api gateway will add the default domain name: https://github.com/cnvergence/api-gateway/blob/main/internal/helpers/default_domain.go